mne_bids/read.py

```diff
@@ -320,7 +320,7 @@ def get_head_mri_trans(bids_fname, bids_root):
     mri_landmarks = mri_landmarks * 1e-3
 
     # Get MEG landmarks from the raw file
-    raw = read_raw_bids(bids_fname, bids_root)
+    raw = read_raw_bids(bids_fname, bids_root, extra_params=dict(allow_maxshield=True))
```

Comment:
This will break for non fif files. Please add the extra_params to function to avoid a bug with non fif files